js/glowingbear.js

```diff
@@ -599,12 +599,8 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
                 if (moreLines) {
                     // We fetched more lines, keep the scroll position constant
                     bl.scrollTop = bl.scrollHeight - bl.clientHeight - sVal;
-                } else {
-                    // New message, scroll with buffer (i.e. to bottom)
-                    var eob = document.getElementById("end-of-buffer");
-                    eob.scrollIntoView();
```

Comment:
this basically does nothing that passing `true` to the subsequent `updateBufferBottom` call doesn't, so do that instead.